package.json

```diff
@@ -11,6 +11,7 @@
   "files": [
     "lib-es5/*.js",
     "lib-es5/*.ts",
+    "lib-es5/*.json",
```

Comment:
Is there an extra tab here?

Reply:
Not sure, shouldn’t it be picked up by linter otherwise?

Reply:
BTW I’m from phone now, just switched off my pc

Reply:
it's fine, it was already indented, but in the diff on the github app everything _but_ this line was left justified 🙄